Description: Fix some Makefile quirks
Author: Ximin Luo <infinity0@pwned.gg>
Bug: https://github.com/sackmotion/motion/pull/16

diff --git a/conf.c b/conf.c
index 2da0da6..4383101 100644
--- a/conf.c
+++ b/conf.c
@@ -1894,7 +1894,7 @@ struct context **conf_load(struct context **cnt)
         fp = fopen(filename, "r");
 
         if (!fp) {
-            snprintf(filename, PATH_MAX, "%s/motion.conf", sysconfdir);
+            snprintf(filename, PATH_MAX, "%s/motion/motion.conf", sysconfdir);
             fp = fopen(filename, "r");
 
             if (!fp) /* There is no config file.... use defaults. */
@@ -2369,6 +2369,6 @@ static void usage()
     printf("-h\t\t\tShow this screen.\n");
     printf("\n");
     printf("Motion is configured using a config file only. If none is supplied,\n");
-    printf("it will read motion.conf from current directory, ~/.motion or %s.\n", sysconfdir);
+    printf("it will read motion.conf from current directory, ~/.motion or %s/motion.\n", sysconfdir);
     printf("\n");
 }
-- 
1.9.0

